scripts/install/roles/env/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 - name: "Install new version (zsh)"
   lineinfile:
     dest: "{{ ansible_env.HOME }}/.zshrc"
-    line: "source /opt/aeriscloud/scripts/env.sh"
+    line: "source {{ install_dir }}/scripts/env.sh"
```

Comment:
Are you sure this won't break anything else? Should there be a `default: /opt` or something?